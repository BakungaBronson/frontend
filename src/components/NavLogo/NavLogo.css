.Logo{
    width: 300px;
    padding: 0px 0px;
}
.Brand img{
    vertical-align: middle;
    border-style: none;
    box-sizing: none;
    height: 68px;

}
.Brand {
    position: absolute;
    /* float: left; */
    line-height: 65px;
    text-transform: uppercase;
    font-size: 1.4em;
    display: inline-block;
    font-size: 22px;
    width: 350px;
    background-color: #008ac1;
}
