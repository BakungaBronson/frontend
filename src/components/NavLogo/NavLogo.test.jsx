import React from 'react';

import { shallow } from 'enzyme';

import NavLogo from '.';


describe('NavLogo Component', () => { 
    it('Renders correctly', () => {
        const NavLogoComponent = shallow(<NavLogo />)
        expect(NavLogoComponent).toMatchSnapshot();
        expect(NavLogoComponent.find('img').length).toBe(1)
    })
})