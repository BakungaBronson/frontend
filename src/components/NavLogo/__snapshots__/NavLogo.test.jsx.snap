// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLogo Component Renders correctly 1`] = `ShallowWrapper {}`;
